introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<p>I would extend the link so that it says "Amazon SageMaker <a href="https://sagemaker.readthedocs.io/en/stable/algorithms/index.html" target="_blank">Built-in Algorithms</a>" instead of just "<a href="https://sagemaker.readthedocs.io/en/stable/algorithms/index.html" target="_blank">Built-in Algorithms</a>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done.</p>